--- src/compat/compat.h	2021-08-10 19:15:01.000000000 +0100
+++ src/compat/compat.h	2021-08-10 19:15:02.000000000 +0100
@@ -39,10 +39,6 @@
 #error "WireGuard requires Linux >= 3.10"
 #endif

-#if LINUX_VERSION_CODE >= KERNEL_VERSION(5, 6, 0)
-#error "WireGuard has been merged into Linux >= 5.6 and therefore this compatibility module is no longer required."
-#endif
-
 #if defined(ISRHEL7)
 #include <linux/skbuff.h>
 #define headers_end headers_start